packages/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const ordersController = require('../controllers/orders.controller');
+
+router.get('/:id', (req, res) => {
+  const userID = req.params.id;
+  const pattern = /^\d+$/;
+
+  if (!userID.match(pattern)) {
```

Comment:
Should we not be checking against the existing users in the database?

Reply:
I have moved function that checking userID format orders controller instead and add validation if user ID is exist in database by write mysql query to get users with that userID